CHANGELOG.adoc

```diff
@@ -0,0 +1,275 @@
+= Revision History
+:toc:
+
+[[v1.3.3-sep-1-2016]]
+== 1.3.3 September 1, 2016
```

Comment:
@budhrg we also did this in the main ADB repo, it's best practice to spell out months in docs and it's a part of the evolution of the doc set.